data/_deprecated/deprecated ships.txt

```diff
@@ -212,9 +212,9 @@ ship "Korath World-Ship"
 		hull 34000
 		"required crew" 794
 		"bunks" 1492
-		"mass" 1735
-		"drag" 21
-		"heat dissipation" .4
+		"mass" 4440
+		"drag" 43
+		"heat dissipation" .2
```

Comment:
If we're keeping them around, we ought to keep them consistent with across-the-board changes to current content too, I'd say.

Reply:
I thought it was just for old plugins and such
which if they still use those the updates wouldnt have much sense for them

Reply:
I'll wait and see what the devs think.

Reply:
Alternatively, I could rescale the masses of deprecated ships to preserve their prior handling, rather than updating them as in the PR? I don't think leaving them unchanged is a good idea, since because of the engine changes that'd make them 50% faster.

Reply:
thing is we're not going to change any ships acquired prior to the PR anyway
I mean change it if you want

Reply:
At this point we're getting into the question of what exactly `deprecated ships.txt` is for - it's not needed to avoid breaking already-acquired ships, it's only there for backwards compatibility with missions in plugins. I figure if it's worth keeping it around, it's also worth keeping it updated.

Reply:
I was generally of the opinion that the point was to keep old ships as they were without any significant changes. If it's a change that might affect how a ship is outfitted or performs, then it shouldn't be happening to deprecated ships. Either of those risks making ships in plugins broken, and avoiding that is the entire point of deprecated ships.

The point of deprecated ships is to lock their stats at the point when the replaced ship diverges sufficiently from the original to risk stuff based on it breaking. Once forked, it should no longer be changed baring mechanical changes to the game that render it utterly non-functional. (ex. If we changed heat mechanics such that this ship would just take off and fry instantly and permanently, then that would merit a change; and even then I would say that it should be the minimum change required to allow it to perform the same as it did prior to the mechanical change, even if that differs significantly from how the parallel vanilla ship performs.

tldr: Deprecated ships are locking the ship to that point in time, and the only changes that should happen to it are ones that keep it functioning as close to exactly as it was at the point in time it was deprecated in the face of changes to the mechanics. Nothing else.

Reply:
That's a reasonable point. In which case, I'll update this accordingly, to preserve prior performance of deprecated ships as it would have been without the engine tweak.